app/src/main/java/com/duckduckgo/app/global/DuckDuckGoActivity.kt

```diff
@@ -45,5 +54,12 @@ abstract class DuckDuckGoActivity : AppCompatActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    override fun onDestroy() {
+        themeChangeReceiver?.let {
+            LocalBroadcastManager.getInstance(applicationContext).unregisterReceiver(it)
```

Comment:
We could inject in the local broadcast manager, so we don't have to keep grabbing it by the `getInstance` method.

    @Inject
    lateinit var broadcastManager: LocalBroadcastManager

If we did do that, we could simplify the `Theming.kt` extension methods too, if we extended `DuckDuckGoActivity` instead of `Activity`. For example, 

before:

    fun Activity.sendThemeChangedBroadcast() {
        val manager = LocalBroadcastManager.getInstance(applicationContext)
        manager.sendBroadcast(Intent(BROADCAST_THEME_CHANGED))
    }

would become:

    fun DuckDuckGoActivity.sendThemeChangedBroadcast() {
        broadcastManager.sendBroadcast(Intent(BROADCAST_THEME_CHANGED))
    }

and so on for all the other theming extension functions... not required, but since the theming extensions only apply to `DuckDuckGoActivity` (i think?), we should probably tighten the scope of the extension methods anyway. so if we did do that, we could access the already injected local broadcast manager from `DuckDuckGoActivity`. 

Not sure if that ramble makes sense... it does in my head. 🙃